README.md

```diff
@@ -118,3 +118,20 @@ Once the repository has been created, the CI stored in this template (`.github/w
 ### Clone the repo to your local development environment
 
 Once you have set your repo up, you can clone it to your local development environment using `git clone [repo address]` or a UI/IDE tool, if preferred.
+
+### Dependency Management with Renovate
+
+Out of the box this repo uses [Renovate](https://docs.renovatebot.com/) for dependency management. More information on the decision to try renovate can be found [here](https://github.com/navapbc/template-application-nextjs/blob/main/docs/decisions/0002-use-renovate-for-dependency-updates.md). Renovate is free and open-source and allows us to bundle dependency updates together and customize their scheduling.
+
+**Getting started with renovate**:
+1. Install Renovate's GitHub App for your repo ([Docs](https://docs.renovatebot.com/getting-started/installing-onboarding/#hosted-githubcom-app)). For most projects, you most likely only want to do this for your select repository.
```

Comment:
This is one downside to using Renovate, since it requires teams to request permission to install it on their repo, whereas Dependabot is available out of the box. Based on our experience so far with OCIO on DOL, I don't know if I'd spend energy trying to get them to enable it for a prototype repo, but I could see it being a worthwhile discussion for something that'll need maintained longer term.

Since it requires this step to enable automated dependency PR's, I'm wondering if we should include the original `.dependabot.yml` approach, and the steps to enable Renovate then include a step to remove that file? I don't feel strongly about this, but curious what reactions are to it.

Reply:
I did include a note about this at the end in case that's helpful for folks - like how to just not use renovate and a link to an example .dependabot.yml file to add back!

Reply:
yeah that sounds like a downside.

is renovate free at least? not requiring a procurement could reduce the barrier by quite a bit.

Reply:
Don't have strong feelings or much to add, but I do think the `Opting out of renovate` section helps address the concerns. We could consider moving that section to line 125 here? After we introduce what renovate is and why we picked it, but before we list the steps for using it?

Reply:
@lorenyu - yes renovate is free and open source! @jimmyfagan good idea about moving up the opt out section :) 

I'm also going to add some references about other ways to set up renovate ( you can also set up self-hosting for example)